README.rst

```diff
@@ -25,9 +25,9 @@ The following example shows the real directory structure of some Symfony recipes
                 manifest.json
         framework-bundle/
             3.3/
-                etc/
+                config/
                 src/
-                web/
+                public/
```

Comment:
should be moved before `src` to be consistent with the display of filesystem explorers (they generally sort folders by name)